concordium-std-derive/src/lib.rs

```diff
@@ -627,6 +633,9 @@ fn init_worker(attr: TokenStream, item: TokenStream) -> syn::Result<TokenStream>
     let parameter_option = init_attributes.optional.parameter;
     let return_value_option = None; // Return values are currently not supported on init.
     let error_option = init_attributes.optional.error;
+    let event_option = init_attributes.event;
+    let contract_name = wasm_export_fn_name.clone();
```

Comment:
With respect to my comment in the concordium-base PR, the `init_` comes from this it seems. 

We could in principle remove it here. But I think it's OK the way it is.